rskj-core/src/main/java/co/rsk/rpc/modules/trace/TraceModuleImpl.java

```diff
@@ -54,6 +53,8 @@ public class TraceModuleImpl implements TraceModule {
     private static final String PENDING_BLOCK = "pending";
     private static final Logger logger = LoggerFactory.getLogger("web3");
 
+    private static final int MAX_TRACES_PER_REQUEST = 10000;
```

Comment:
nit: Should this be configurable? if so we should use the `.conf` files instead. We also have a `Constants` file for more internal configurations. Not sure if any of these options are ok for this work.

This is because we kind of try to have all constants in one place